src/components/policy/policy_external/include/policy/policy_manager_impl.h

```diff
@@ -608,6 +608,12 @@ class PolicyManagerImpl : public PolicyManager {
    */
   void OnAppsSearchCompleted(const bool trigger_ptu) OVERRIDE;
 
+  /**
+   * @brief Change applicatios count ready for PTU
```

Comment:
@ydementieiev typo

Reply:
@AKalinich-Luxoft done in [fe7b393](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/fe7b393fe80f89ac9137d6dd9b41a0dfd6ba28c6)